README.md

```diff
@@ -60,10 +62,28 @@ docker \
     -P \
     writl/pyload:latest
 ```
+Sample compose file
+-----
+```yaml
+version: "2"
+services:
+    pyload:
+      image: writl/pyload:latest
+      container_name: pyload
+      network_mode: host
```

Comment:
Why are you using network_mode: host?
I guess this is not needed and should be avoided. Can you remove it?